Changelog.md

```diff
@@ -48,19 +148,10 @@
 * Updated Hellfire Penetrator (#135)
 * Move engine simulation to be local but globally synchronised. (#134)
 
-*15/11/2021*
-* Moved sight select to shift
-
-*07/11/2021*
```

Comment:
Why were these removed?

Reply:
Because they were not ors but commits from prs